src/chart_types/xy_chart/state/selectors/get_tooltip_values_highlighted_geoms.ts

```diff
@@ -196,11 +202,21 @@ function getTooltipAndHighlightFromValue(
     header = null;
   }
 
+  // @ts-ignore
+  const tooltipSortFn = getTooltipCompareFn(settings.sortSeriesBy, (a, b) => {
```

Comment:
This could cause other issues to be ignored.

```suggestion
  const tooltipSortFn = getTooltipCompareFn((settings as any).sortSeriesBy, (a, b) => { 
```